### PR TITLE
Adds clarification about checking in package-lock.yml

### PR DESCRIPTION
**Pull request by ernestoongaro**
internal discussion https://dbt-labs.slack.com/archives/C05CJT5MJF2/p1716376487380389

### CODE REVIEW

File: website/docs/reference/commands/deps.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Update your versions in packages.yml, then run dbt deps
...
 
-Starting in dbt Core v1.7, dbt generates a `package-lock.yml` file in the root of your project. This contains the complete set of resolved packages based on the `packages` configuration in `dependencies.yml` or `packages.yml`. Each subsequent invocation of `dbt deps` will install from the _locked_ set of packages specified in this file. Storing the complete set of required packages (with pinned versions) in version-controlled code ensures predictable installs in production and consistency across all developers and environments.
+Starting in dbt Core v1.7, dbt generates a `package-lock.yml` file in the root of your project. This contains the complete set of resolved packages based on the `packages` configuration in `dependencies.yml` or `packages.yml`. Each subsequent invocation of `dbt deps` will install from the _locked_ set of packages specified in this file. Storing the complete set of required packages (with pinned versions) in version-controlled code ensures predictable installs in production and consistency across all developers and environments. It should be checked into git, and updated every time you make a change to your `packages.yml` file.
```
Comment (b-per) [inline website/docs/reference/commands/deps.md]: It is not so much a change in `packages.yml` is made but more when someone does a `dbt deps --upgrade`  or `dbt deps --lock`. (see the following paragraph)

Comment (mirnawong1) [inline website/docs/reference/commands/deps.md]:
```suggestion
Starting in dbt Core v1.7, dbt generates a `package-lock.yml` file in the root of your project. This contains the complete set of resolved packages based on the `packages` configuration in `dependencies.yml` or `packages.yml`. Each subsequent invocation of `dbt deps` will install from the _locked_ set of packages specified in this file. Storing the complete set of required packages (with pinned versions) in version-controlled code ensures predictable installs in production and consistency across all developers and environments. 

The `package-lock.yml` file should be committed in Git initially, and then updated and committed only when you want to change versions or uninstall a package (for example  `dbt deps --upgrade` or `dbt deps --lock`).
```

Comment (mirnawong1) [reply website/docs/reference/commands/deps.md]: @ernestoongaro @b-per changed it to the above and added the 'uninstall a package ' -- does this look right?

Comment (ernestoongaro) [reply website/docs/reference/commands/deps.md]: Thanks, updated!

Comment (ernestoongaro) [reply website/docs/reference/commands/deps.md]: @b-per are you happy with what Mirna updated? https://github.com/dbt-labs/docs.getdbt.com/pull/5544/commits/27b05c4b8e8956f044f71ca774d5ac4863cfc70f

**Review by b-per — APPROVED** [review]

**Comment by mirnawong1** [comment]
thanks ya'll, checking for the failure

**Review by mirnawong1 — APPROVED** [review]

**Comment by mirnawong1** [comment]
@ernestoongaro @b-per the failure is coming from the role permissions but the preview is looking fine in my localhost. will merge and look at the permissions after if that's ok!

Screenshot 2024-05-23 at 14 15 14

**Comment by ernestoongaro** [comment]
TYSM @mirnawong1 ! You are the best :)

**Comment by mirnawong1** [comment]
this is live now ya'll: https://docs.getdbt.com/reference/commands/deps#predictable-package-installs thank you so much @ernestoongaro and @b-per !

**Pull request merged**